connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -803,9 +836,13 @@ message ReleaseSessionRequest {
   optional string client_type = 3;
 }
 
+// Next ID: 3
 message ReleaseSessionResponse {
   // Session id of the session on which the release executed.
   string session_id = 1;
+  // Server-side generated idempotency key that the client can use to assert that the server side
+  // session has not changed.
+  string server_side_session_id = 2;
```

Comment:
in this special case, should it be optional, because it will not be set when the session doesn't exist?
will your wrapper handle that?

Reply:
Yes, the wrapper will ignore the null server_side_session_id as it is identical to being backwards compatible to older servers.

Reply:
it will not be null but empty string I believe, as "" is the default value for primitive string type?

Reply:
hm, wouldn't it come as empty string from older servers then as well?

Reply:
It is "" and I checked that the code properly deals with ""